src/nemo_evaluator/observability/model_traffic.py

```
@@ -2,13 +2,30 @@
 from __future__ import annotations
 
+import hashlib
 import json
 import threading
 import time
 import uuid
 from collections import Counter
 from typing import TYPE_CHECKING, Any, Iterator
 
+
+def _request_hash(body: Any) -> str | None:
```

Ruff (0.15.17)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `body`

(ANN401)